engine/src/test/java/com/datatorrent/stram/plan/physical/PhysicalPlanTest.java

```diff
@@ -2246,4 +2253,97 @@ public void testParallelPartitionForSlidingWindow()
     PhysicalPlan plan = new PhysicalPlan(dag, new TestPlanContext());
     Assert.assertEquals("number of containers", 7, plan.getContainers().size());
   }
+
+  static class NoOpStatsListener implements StatsListener
```

Comment:
I have use Mockito to mock the interface in the tests, and updated the pull request